client/src/pages/private/PSIViewDetails.js

```diff
@@ -160,7 +160,7 @@ export default ({ match }) => {
       <Page>
         <CheckPermissions permittedRoles={['ministry_of_health']} renderErrorMessage={true}>
           <Card>
-            <Box pt={4} pb={2} pl={4} pr={4}>
+            <Box pt={4} pb={2} pl={4} pr={4} width={400}>
```

Comment:
I'm not sure this is big enough to prevent layout shifts based on changing data, but this is a larger layout issue that can possible be addressed separately, the same issue affects many pages